.github/workflows/quickstart_opentelemetry_ci.yml

```diff
@@ -0,0 +1,16 @@
+
+name: WildFly OpenTelemetry Quickstart CI
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths:
+      - 'opentelemetry/**'
```

Comment:
this should match the QS path, i.e. opentelemetry-tracing

Reply:
Fixed.